src/main/java/com/google/sps/data/Attendee.java

```diff
@@ -0,0 +1,72 @@
+package com.google.sps.data;
+
+import java.util.Date;
+import com.google.appengine.api.datastore.Entity;
+
+/** Class that represents a user in the session as an attendee. */
+public class Attendee implements AttendeeInterface{
+
+  // The id of the session this attendee is in.
+  private final String sessionId;
+
+  // The screen name of this attendee.
+  private final String screenName;
+
+  // Time the attendee last polled the server.
+  private Date timeLastPolled;
+  
+  /** Initializes an Attendee object
+   * @param {String} sessionId - the session the attendee is attached to.
+   * @param {String} screenName - the screen name of the attendee. 
+   * @param {Date} timeLastPolled - the time attendee was last polled.
+   */
+  public Attendee (String sessionId, String screenName, Date timeLastPolled) {
+    this.sessionId = sessionId;
+    this.screenName = screenName;
+    this.timeLastPolled = timeLastPolled;
+  }
+
+  public String getSessionId() {
+    return sessionId;
+  }
+
+  public String getScreenName() {
+    return screenName;
+  }
+
+  public Date getTimeLastPolled() {
+    return timeLastPolled;
+  }
+  
+  /**
+   * Returns a new Entity of kind "Attendee" from an Attendee object.
+   * @param {Attendee} attendee - the Attendee object that will be made into
+   *     an Entity.
+   */
+  public Entity toEntity(Attendee attendee) {
+    Entity attendeeEntity = new Entity("Attendee");
+    attendeeEntity.setProperty(this.sessionId, attendee.sessionId);
+    attendeeEntity.setProperty(this.screenName, attendee.screenName);
```

Comment:
i think you got the properties mixed up + you could use the entity constants here no?